modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase

```diff
@@ -22,6 +22,74 @@ definition {
 		}
 	}
 
+	@description = "LPS-149508 - Verify CRUD functionality with audit when using object definition"
+	@priority = 5
+	test AssertCRUDWithObjectDefinition {
+		property portal.acceptance = "true";
+
```

Comment:
Let's not include this as an acceptance test until we're sure of the stability of the test.